app/src/main/java/com/ddoczi/tasky/agenda/presentation/detail/task/TaskDetailScreen.kt

```diff
@@ -1,9 +1,91 @@
 package com.ddoczi.tasky.agenda.presentation.detail.task
 
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Divider
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.ddoczi.tasky.R
+import com.ddoczi.tasky.agenda.presentation.detail.composables.DetailColor
+import com.ddoczi.tasky.agenda.presentation.detail.composables.DetailDescription
+import com.ddoczi.tasky.agenda.presentation.detail.composables.DetailHeader
+import com.ddoczi.tasky.agenda.presentation.detail.composables.DetailTitle
+import com.ddoczi.tasky.core.presentation.composables.TaskyBackground
+import com.ddoczi.tasky.ui.theme.Gray
+import com.ddoczi.tasky.ui.theme.Green
+import com.ddoczi.tasky.ui.theme.Light
 
 @Composable
-fun TaskDetailScreen() {
-    Text(text = "Task Detail Screen")
+fun TaskDetailScreen(
+    state: TaskDetailState,
+    onEvent: (TaskDetailEvent) -> Unit
+) {
```

Comment:
Given that all agenda item detail screens have quite a lot in common, could it make sense to have one agenda detail screen instead? Especially the first half is pretty much the same for each item.
You can of course still extract the sections into separate components to keep them reusable and readable